doc/use_cases/turtlebot3_tutorial.md

```diff
@@ -0,0 +1,239 @@
+
```

Comment:
` doc/use_cases/turtlebot3_tutorial.md ` this really doesn't belong in use cases, can you make a new tutorials folder and move the slam tutorial into that as well?

Reply:
sounds good. I will do these changes in this PR as well.

Reply:
I moved this file to tutorials directory. I think Nav2 with Slam belongs in use cases. It is a very generic documentation. In my opinion, Nav2 with Slam Toolbox or Cartographer can go into tutorials.